tests/standalone/run_all.py

```diff
@@ -301,6 +304,24 @@ def checkRequirements(filename):
     return (True, "")
 
 
+def ParseCommandLineArguments(filename):
+    '''
+	Checks if the program requires command line arguments and parses
+	the arguments provided.
+    The command line arguments should be presented in the form of a
+    string assigned to a variable named commands. The assignment should be in the form: 
+	    commands='command1 command2=87 command3 --command4'
+	The commands may be positional or optional and may or may not be
+	assigned to values, as the case may be.
+	'''
+    for line in getFileContentByLine(filename):
+        if line.strip().startswith('commands'):
+            parser=argparse.ArgumentParser()
+            parser.parse_args(commands.split()) # At this point, the global namespace
```

Comment:
Preferable formatting:
```
# Comment line
# Comment line
# Comment line
<python-statement>
```
In this case though, the comment can follow after the statement, but should not start on the statement's line itself.

Reply:
okay. Thanks. I just learnt that now. it will be in my next commit